assets/src/components/minimalLadderPage.tsx

```diff
@@ -1,36 +1,26 @@
 import React, { useContext } from "react"
-import RouteLadders from "./routeLadders"
-import { currentRouteTab } from "../models/routeTab"
+import { isPreset } from "../models/routeTab"
 import { StateDispatchContext } from "../contexts/stateDispatchContext"
-import {
-  deselectRouteInTab,
-  flipLadderInTab,
-  toggleLadderCrowdingInTab,
-} from "../state"
+import { Button } from "react-bootstrap"
 
 export const MinimalLadderPage = () => {
-  const [{ routeTabs }, dispatch] = useContext(StateDispatchContext)
-
-  const { selectedRouteIds, ladderDirections, ladderCrowdingToggles } =
-    currentRouteTab(routeTabs) || {
-      selectedRouteIds: [] as string[],
-      ladderDirections: {},
-      ladderCrowdingToggles: {},
-    }
+  const [{ routeTabs }] = useContext(StateDispatchContext)
+  const presets = routeTabs
+    .filter(isPreset)
+    .sort((a, b) => (a.presetName || "").localeCompare(b.presetName || ""))
 
   return (
-    <div className="c-ladder-page__tab-bar-and-ladders">
-      <RouteLadders
-        selectedRouteIds={selectedRouteIds}
-        selectedVehicleId={undefined}
-        deselectRoute={(routeId) => dispatch(deselectRouteInTab(routeId))}
-        reverseLadder={(routeId) => dispatch(flipLadderInTab(routeId))}
-        toggleCrowding={(routeId) =>
-          dispatch(toggleLadderCrowdingInTab(routeId))
-        }
-        ladderDirections={ladderDirections}
-        ladderCrowdingToggles={ladderCrowdingToggles}
-      />
+    <div className="c-minimal-ladder-page">
+      <h3>Select a preset to display</h3>
+      {presets.map((preset) => (
+        <Button
+          className="c-minimal-ladder-page__button"
+          key={preset.uuid}
+          href={"/minimal/" + preset.uuid}
+        >
+          {preset.presetName}
+        </Button>
+      ))}
```

Comment:
suggestion: Can we use bootstrap to style and lay these out?
```suggestion
      <Nav>
        {presets.map((preset) => (
          <Nav.Item key={preset.uuid}>
            <Nav.Link as={Link} to={"/minimal/" + preset.uuid}>
              {preset.presetName}
            </Nav.Link>
          </Nav.Item>
        ))}
      </Nav>
```

Reply:
Hmmmm I've waffled a bit. My default inclination was to use buttons, and I'm not married to that. But they _are_ bootstrap, and I aesthetically like them. It feels nitpicky to not just commit the above, but since I have to add the imports anyway, I think I'm going to commit a blend that I have, as noted and seen in [this comment](https://github.com/mbta/skate/pull/2588#discussion_r1599100828), but I'll keep the Link

Reply:
I think this is related? https://github.com/mbta/skate/pull/2588#discussion_r1603608483